### PR TITLE
Allow configuration of `searchType` by site

### PR DESCRIPTION
**Pull request by Shinsina**
**REQUIRES https://github.com/parameter1/base-browse/pull/57**

Screenshot from 2023-06-06 11-04-25

### CODE REVIEW

**Review by solocommand — CHANGES_REQUESTED** [review]
Since `useAutocomplete` is already in production, don't remove it to prevent backwards breaking behavior. Only add the new simple search

File: packages/marko-web-search/components/query.marko [hunk 1]
```diff
@@ -22,7 +22,7 @@ $ const params = {
   ),
   queryFragment: input.queryFragment,
   opSuffix: input.opSuffix,
-  useAutocomplete: site.get('search.useAutocomplete', false),
+  useMatchPhrase: site.get('search.useMatchPhrase', false),
```
Comment (zarathustra323) [inline packages/marko-web-search/components/query.marko]: `searchType: site.get('search.searchType'),`

File: packages/marko-web-search/loaders/search.js [hunk 2]
```diff
@@ -64,16 +64,16 @@ module.exports = async ({ apolloBaseCMS, apolloBaseBrowse } = {}, {
...
   const input = {
     omitScheduledAndExpiredContent: true,
     statuses: ['PUBLISHED'],
     contentTypes,
     countryCodes,
-    ...(!useAutocomplete && searchQuery && { search: { query: searchQuery } }),
-    ...(useAutocomplete && searchQuery && { autocomplete: { phrase: searchQuery } }),
+    ...(!useMatchPhrase && searchQuery && { search: { query: searchQuery } }),
+    ...(useMatchPhrase && searchQuery && { search: { query: searchQuery, type: 'MATCH_PHRASE' } }),
     ...((assignedToWebsiteSiteIds.length || assignedToWebsiteSectionIds.length) && {
```
Comment (zarathustra323) [inline packages/marko-web-search/loaders/search.js:76]: `...(searchQuery && { search: { query: searchQuery, type: searchType } }),`

File: packages/marko-web-search/loaders/search.js [hunk 3]
```diff
@@ -64,16 +64,16 @@ module.exports = async ({ apolloBaseCMS, apolloBaseBrowse } = {}, {
 
   queryFragment,
   opSuffix,
-  useAutocomplete = false,
+  useMatchPhrase = false,
```
Comment (zarathustra323) [inline packages/marko-web-search/loaders/search.js]: `searchType,`

**Review by zarathustra323 — CHANGES_REQUESTED** [review]

**Review by zarathustra323 — APPROVED** [review]

**Comment by Shinsina** [comment]
[Search: Enable Exact Phrase Search](https://trello.com/c/3VXxkC6f/451-search-enable-exact-phrase-search)

**Pull request merged**